### PR TITLE
Update become-a-full-node-operator-from-source.md

### PR DESCRIPTION
**Pull request by iammortimer**
Fixed link to go install documentation, updated version

### CODE REVIEW

File: nodes-validators/networks/mainnet/become-a-full-node-operator-from-source.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This tutorial assumes that your node is running Ubuntu LTS version (i.e: 18.04,
 ### Go version (required)
 
 The Golang version should be from 1.18 and above
-If you have not installed it yet, you can refer to [this document](https://github.com/oraichain/docs/blob/master/developer/tutorials/install-go.md).
+If you have not installed it yet, you can refer to [this document]((https://github.com/iammortimer/docs/blob/master/nodes-validators/tutorials/install-go.md)).
```
Comment (ducphamle2) [inline nodes-validators/networks/mainnet/become-a-full-node-operator-from-source.md:16]: Do you mind changing the url to this location instead: https://github.com/oraichain/docs/blob/master/nodes-validators/tutorials/install-go.md

**Review by ducphamle2 — CHANGES_REQUESTED** [review]

**Review by iammortimer — COMMENTED** [review]
Fixed.

File: nodes-validators/networks/mainnet/become-a-full-node-operator-from-source.md (hunk 1 above)
Comment (iammortimer) [reply nodes-validators/networks/mainnet/become-a-full-node-operator-from-source.md]: Apologies, copied the wrong link.

**Comment by ducphamle2** [comment]
lgtm!

**Pull request merged**